RadixWallet/Clients/PreAuthorizationClient/PreAuthorizationClient+Interface.swift

```diff
@@ -29,6 +29,6 @@ extension PreAuthorizationClient {
 
 	struct PollStatusRequest: Sendable {
 		let subintentHash: SubintentHash
-		let expiration: DappToWalletInteractionSubintentExpiration
+		let expirationTimestamp: Instant
 	}
```

Comment:
I will not push for more optimizations, we can do it later, but this should not be needed. 
The WalletToDappInteractionSubintentResponseItem contains the full Subintent, which can then be sent to Sargon and it would extract the proper timestamp and hash. This API sort of allows having an unrelated timestamp to a given subintent hash.

Reply:
Yes, in fact sending the entire `Subintent` rather than the `SubintentHash` and `Instant` is what I proposed on SU yesterday.

However, we discussed it further with @sergiupuhalschi-rdx and agreed to keep it this way to avoid sending unnecessary information (the more data we send over UniFFI the slowest right?) and making more updates on both sides. We didn't consider the fact that this allows having an unrelated timestamp, though